src/pages-conditional/developers/local-environment.js

```diff
@@ -605,30 +625,105 @@ export const query = graphql`
         }
       }
     }
+    waffle: file(relativePath: { eq: "devtools/waffle.png" }) {
+      ...devtoolImage
+    }
+    waffleGitHub: github {
+      repository(owner: "EthWorks", name: "waffle") {
```

Comment:
Nice work pulling these in via GraphQL! FYI we could also save manually tracking the `gitHubUrl`s above by adding `url` key within these queries, e.g.
```
  waffleGitHub: github {
    repository(owner: "EthWorks", name: "waffle") {
      stargazerCount
      languages(orderBy: {field: SIZE, direction: DESC}, first: 2) {
        nodes {
          name
        }
      }
      url
    }
  }
```